__mocks__/handlers/tasks.handler.js

```diff
@@ -626,6 +626,55 @@ const taskHandlers = [
 	rest.patch(`${URL}/tasks/:taskId`, (_, res, ctx) => {
 		return res(ctx.delay(5000), ctx.status(204));
 	}),
+	rest.post(`${URL}/tasks`, async (req, res, ctx) => {
+		const body = await req.json();
+		return res(
+			ctx.status(200),
+			ctx.json({
+				message: 'Task created successfully!',
+				task: body
+			})
+		);
+	})
 ];
 
+export const failedGetTasksResponse = {
+	error: 'Internal Server Error',
+	message: 'An internal server error occurred',
+	statusCode: 500
+};
+
+export const failedGetTasks = rest.get(
+	`${URL}/tasks`,
+	(_, res, ctx) => {
+		return res(ctx.status(500), ctx.json(failedGetTasksResponse));
+	}
+);
+
+export const failedAddNewTaskResponse = {
+	'statusCode': 400,
+	'error': 'Bad Request',
+	'message': '"title" is required'
+};
+export const failedAddNewTaskHandler = rest.post(
+	`${URL}/tasks`,
+	(_, res, ctx) => {
+		return res(ctx.status(404), ctx.json(failedAddNewTaskResponse)
+		);
```

Comment:
here 404 and above 400??

Reply:
Both are 404 only right?

Reply:
404- not found
400 - Bad request

Reply:
No, I'm not telling that. I'm saying in both the files, I have used `404` isn't it?
`expect(nextResponse.error).toHaveProperty('status', 404);`
`return res(ctx.status(404), ctx.json(failedAddNewTaskResponse)`

Reply:
<img width="452" alt="image" src="https://github.com/Real-Dev-Squad/website-status/assets/107163260/6874b224-dbba-444d-834f-e66b27522edb">
Hi @bharati-21 this both I am talking about failedAddNewTaskResponse, failedAddNewTaskHandler

Reply:
I understood. I'll make this change. This is supposed to be 400 only. Thank you for pointing this out.

Reply:
Made the changes